src/NextGen/DonationForm/ViewModels/DonationFormViewModel.php

```diff
@@ -269,6 +271,29 @@ private function enqueueFormScripts(int $formId, string $formDesignId)
             }
         }
 
+        // load extensions
```

Comment:
Is this duplicated from `src/FormBuilder/Routes/RegisterFormBuilderPageRoute.php`?

Reply:
It's not exactly a duplicate as we call distinct methods based on the current screen.

`RegisterFormBuilderPageRoute.php` uses:
```php
$extension->formBuilderCss();
$extension->formBuilderJs();
$extension->formBuilderDependencies();
```

`DonationFormViewModel.php` uses:
```php
$extension->formViewCss();
$extension->formViewJs();
$extension->formViewDependencies();
```